linq4j/src/main/java/org/apache/calcite/linq4j/tree/Expressions.java

```diff
@@ -70,8 +70,8 @@ public static String toString(List<? extends Node> expressions, String sep,
   /**
    * Converts an expression to Java source code.
    */
-  public static String toString(Node expression) {
-    return toString(Collections.singletonList(expression), "", true);
+  public static String toString(Node expression, boolean methodSplit) {
```

Comment:
Since we are touching public methods of a public class, to facilitate backwards compatibility I wonder if it would make sense to overload these `toString` methods (to have a "default false methodSplit", in case any downstream project was using them directly):
```
public static String toString(List<? extends Node> expressions, String sep,
      boolean generics) {
  return toString(expressions, sep, generics, false);
}

public static String toString(Node expression) {
  return toString(expression, false);
}

```

Reply:
Done in [dc1c66d](https://github.com/apache/calcite/pull/3901/commits/dc1c66d14b38ad122312b99861fca20794701504)